syntax="proto3";
package com.bitcoin.proto3;

import "OutputV3.proto";

// meaningless enum example added for illustration

enum TimeOfDay {
  NotSpecified = 0;
  AM = 1;
  PM = 2;
}

message Payment {
  bytes            merchant_data = 1;
  repeated bytes   transactions  = 2;
  repeated Output  refund_to     = 3;
  string           memo          = 4;
  TimeOfDay        tod           = 6;
}
